src/composed/message/types.rs

```diff
@@ -398,7 +400,21 @@ impl Message {
             } => {
                 if let Some(message) = message {
                     match **message {
-                        Message::Literal(ref data) => signature.verify(key, data.data()),
+                        Message::Literal(ref data) => {
+                            match signature.typ() {
```

Comment:
Instead of two arm match maybe a simpler `if let` would be sufficient?

Reply:
> Instead of two arm match maybe a simpler `if let` would be sufficient?

Note that the outer match is pre-existing code. I don't think this PR should change that code?
I pushed a variation of my change that uses `matches!`, which I guess is slightly nicer to read.